radio/util/tts_en.py

```diff
@@ -91,29 +91,29 @@
                             (u"normal. mode!, on", "nrmmod"),
                             (u"landing. mode!, on", "lnding"),
                             (u"acro. mode!, on", "acro"),
-                            (u"flight. mode!, one", "fm-1"),
-                            (u"flight. mode!, two", "fm-2"),
-                            (u"flight. mode!, three", "fm-3"),
-                            (u"flight. mode!, four", "fm-4"),
-                            (u"flight. mode!, five", "fm-5"),
-                            (u"flight. mode!, six", "fm-6"),
-                            (u"flight. mode!, seven", "fm-7"),
-                            (u"flight. mode!, eight", "fm-8"),
+                            (u"flight mode one", "fm-1"),
```

Comment:
Are we using Python3 or Python2 here? Python3 doesn't need the u prefix, and it would be nice to avoid them on Python2 with `from __future__ import unicode_literals`